foxpuppet/expected.py

```diff
@@ -10,7 +10,7 @@ class new_browser_window_is_opened(object):
     """An expectation for checking that a new window is found.
 
     Returns:
-        A BrowserWindow object.
+        :py:class:`BrowserWindow`: Browser window.
```

Comment:
I still don't like how this looks. I know we want to link the docs together but maybe there is a better way.